blast/formatter/server/internal/server/run-report-ep.go

```diff
@@ -82,6 +84,16 @@ func zipDir(path string) (err error) {
 		return
 	}
 
+	zipPath := filepath.Join(path, reportExportName)
+
+	exists, err := xfiles.FileExists(zipPath)
+	if err != nil {
+		return
+	}
+	if exists {
+		return errors.New(zipPath + " already exists!")
+	}
+
```

Comment:
Check for the existence of the zip file before attempting to create a zip file.  If the zip file exists, fail.